docs/sources/get-started/labels/non-indexed-labels.md

```diff
@@ -0,0 +1,50 @@
+---
+menuTitle: Non-indexed labels
+title: What are non-indexed labels
+description: Attaching metadata to logs.
+---
+# What are non-indexed labels
+
+One of the powerful features of Loki is parsing logs at query time to extract metadata and build labels out of it.
+However, the parsing of logs at query time comes with a cost which can be significantly high for, as an example,
+large json blobs or a poorly written query using complex regex patterns.
+
+In addition, the data extracted from logs at query time is usually high cardinality, which can’t be stored
+in the index without blowing the cardinality.
```

Comment:
"blowing the cardinality" is confusing here. It's not clear what you mean.

Reply:
By _blowing the cardinality_, I mean increasing the cardinality too much.

What about:
```suggestion
in the index as it would increase the cardinality too much, and therefore reduce the performance of the index.
```

Reply:
That's what I thought you probably meant, but I'm not sure everyone would have understood.  The rewording is much clearer.